tests/client/unit/app/services/test.project.service.js

```diff
@@ -54,5 +54,98 @@ describe('project.service', function () {
         // Assert
         expect(taskGrid.length).toBe(23);
     });
+
+    it('should return a VALID result', function(){
```

Comment:
Please explain the tests a bit more: 'e.g. 'should return a VALID result when validating a AOI that is not self intersecting'